personal_access_tokens.go

```diff
@@ -134,3 +134,23 @@ func (s *PersonalAccessTokensService) RevokePersonalAccessToken(token int, optio
 
 	return s.client.Do(req, nil)
 }
+
+// RotatePersonalAccessToken revokes a token and returns a new token that expires in one week.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/personal_access_tokens.html#rotate-a-personal-access-token
+func (s *PersonalAccessTokensService) RotatePersonalAccessToken(token int, options ...RequestOptionFunc) (*PersonalAccessToken, *Response, error) {
+	u := fmt.Sprintf("personal_access_tokens/%d/rotate", token)
+	req, err := s.client.NewRequest(http.MethodPost, u, nil, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	pat := new(PersonalAccessToken)
+	resp, err := s.client.Do(req, pat)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return pat, resp, nil
+}
```

Comment:
Can you place this method above `RevokePersonalAccessToken` so it matches the ordering in the docs?

Reply:
Fixed in new push.